src/components/LeaderboardTable.js

```diff
@@ -6,6 +6,14 @@ import LeaderboardHandle from "./LeaderboardHandle";
 const LeaderboardTable = ({ results }) => {
   const columns = React.useMemo(
     () => [
+      {
+        Header: 'Rank',
+        Cell: (props) => {
+          return (
+            <p>{props.row.original.rank}.</p>
+          )
```

Comment:
Sorry for the nit-picky edits; it's a good time to point out some styles that we've been adopting.

Like semi-colons.

```suggestion
          );
```